x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/cluster/routing/allocation/DataTierIT.java

```diff
@@ -33,13 +33,42 @@
         return Collections.singleton(LocalStateCompositeXPackPlugin.class);
     }
 
-    public void testDefaultAllocateToHot() {
+    public void testDefaultIndexAllocateToContent() {
```

Comment:
I believe `DataTierIT` needs to be moved under `internalClusterTest` as otherwise, the tests don't run

Reply:
good catch!